tox.ini

```diff
@@ -25,6 +25,25 @@ passenv =
 commands =
     coverage run --parallel --source h,tests/h -m pytest {posargs:tests/h/}
 
+[testenv:dev]
+deps = -rrequirements-dev.in
+passenv =
+    ALLOWED_ORIGINS
+    AUTHORITY
+    BOUNCER_URL
+    CLIENT_OAUTH_ID
+    CLIENT_RPC_ALLOWED_ORIGINS
+    CLIENT_URL
+    CONFIG_URI
+    MODEL_CREATE_ALL
+    SEARCH_AUTOCONFIG
+    USE_HTTPS
+    WEBSOCKET_URL
```

Comment:
And now we have an alphabetically sorted list of all the envvars that can possibly be in use, when `make dev` is being run.